Userland/Libraries/LibJS/Tests/modules/basic-modules.js

```diff
@@ -210,6 +210,10 @@ describe("in- and exports", () => {
     test("can have top level using declarations which trigger at the end of running a module", () => {
         expectModulePassed("./top-level-dispose.mjs");
     });
+
+    test("can export default a RegExp", () => {
+        expectModulePassed("./default-regexp-export.mjs");
```

Comment:
You could add a check to make sure it actually exports the regex see for example the `default exports` test above